app/backend/test/teet/integration/integration_email_test.clj

```diff
@@ -4,12 +4,14 @@
             [teet.test.utils :as tu]
             [clojure.string :as str]))
 
-(def outbox (atom []))
+(def outbox (atom [:server {} :msg {}]))
+
+(defn- update-outbox [server msg] (swap! outbox conj server msg))
```

Comment:
Each message should have server and message, so `(swap! outbox conj {:server server :msg msg})`

Reply:
Fixed.